db/deploy/main.py

```diff
@@ -0,0 +1,76 @@
+import json
+import os
+import subprocess
+from typing import Dict, Literal
+
+from fastapi import FastAPI, HTTPException, Query, Request
+from google.cloud import logging, secretmanager
+
+app = FastAPI()
+
+# Setup for Google Cloud clients and logging
+SECRET_CLIENT = secretmanager.SecretManagerServiceClient()
+LOGGING_CLIENT = logging.Client()
+SECRET_PROJECT = 'sample-metadata'
+SECRET_NAME = 'liquibase-schema-updater'
+log_name = 'lb_schema_update_log'
+changelog_file = 'project.xml'
+logger = LOGGING_CLIENT.logger(log_name)
+
+
+def read_db_credentials(env: Literal['prod', 'dev']) -> Dict[Literal['dbname', 'username', 'password', 'host'], str]:
+    """Get database credentials from Secret Manager."""
+    try:
+        secret_path = SECRET_CLIENT.secret_version_path(SECRET_PROJECT, SECRET_NAME, 'latest')
+        response = SECRET_CLIENT.access_secret_version(request={'name': secret_path})
+        return json.loads(response.payload.data.decode('UTF-8'))[env]
+    except Exception as e:  # Broad exception for example; refine as needed
+        text = f'Failed to retrieve or parse secrets: {e}'
+        logger.log_text(text, severity='ERROR')
+        raise HTTPException(status_code=500, detail=text) from e
+
+
+@app.post('/execute-liquibase')
+async def execute_liquibase(request: Request, environment: Literal['prod', 'dev'] = Query(default='dev', regex='^(prod|dev)$')):
```

Comment:
Is black skipping formatting this file? I expect different formatting here

Reply:
Doesn't look like it's skipping on my end, nothing in the config should cause this folder to be skipped from what I can see